src/moduler/aktivitet/visning/avtalt-container/aktivitet/AvtaltFormContainer.tsx

```diff
@@ -52,6 +54,7 @@ const AvtaltFormContainer = (props: Props) => {
                 className="aktivitetvisning__underseksjon avtalt-container"
                 oppdaterer={bekreftStatus === STATUS.RELOADING}
                 mindreEnnSyvDagerTil={mindreEnnSyvDagerTil}
+                manglerTilDato={manglerTilDato}
```

Comment:
Hva med å heller gjøre sånn her? Da kan du fjerne den linjen med manglerTilDato-konstanten lenger oppe
```suggestion
                manglerTilDato={!aktivitet.tilDato}
```

Reply:
Du er en javascript guru, @theahj 👍